app/response_processor.py

```diff
@@ -106,33 +106,41 @@ def send_receipt(self, decrypted_json):
             receipt_json = {
                 'tx_id': decrypted_json['tx_id'],
                 'collection': {
-                    'exercise_sid': decrypted_json['collection']['exercise_sid']
+                    'exercise_sid':
+                    decrypted_json['collection']['exercise_sid']
                 },
                 'metadata': {
                     'ru_ref': decrypted_json['metadata']['ru_ref'],
                     'user_id': decrypted_json['metadata']['user_id']
                 }
             }
         except KeyError as e:
-            self.logger.error("Unsuccesful publish, missing key values", error=e)
+            self.logger.error(
+                "Unsuccesful publish, missing key values", error=e)
             raise QuarantinableError
 
+        try:
+            case_id = decrypted_json['case_id']
```

Comment:
Not sure about this, because if we get a case id we don't need any of the other information in the receipt json at all. I think it should be a separate json structure completely for RM receipts.

Reply:
Would break receipt-rrm validation? Keeping backwards compatibility for now.